.travis/run_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```

Comment:
As this file only runs tests for Travis, should it have "travis" in the filename?

Reply:
it is in a directory called '.travis'

Reply:
I had not spotted that. :eyeglasses: